roles/network/defaults/main.yml

```diff
@@ -71,3 +71,10 @@ named_enabled: True
 dnsmasq_enabled: False
 dnsmasq_install: False
 captive_portal_enabled: False
+
+# for simple python captive portal
+py_captive_portal_install: True
+py_captive_portal_enabled: True
+captive_portal_port: "9090"
+captive_portal_username: "Admin"
+captive_portal_password: "g0adm1n"
```

Comment:
@m-anish & @jvonau can we change this to Admin/changeme as is used in all other cases?

Currently this username/password creates confusion, breaking with the tradition used by (essentially) all other playbooks.

Reply:
Thx to @jvonau who fixed this from Admin/g0adm1n to Admin/changeme

@m-anish suggests we consider removing the password entirely, "it can just be a simple html page with a button, or as Tim said, a timer"

FYI regardless, @jvonay clarifies this form/pages "adds the iptables rules to allow internet access"